pandas/tests/io/formats/test_console.py

```diff
@@ -0,0 +1,54 @@
+import pytest
+from functools import partial
+
+from pandas.io.formats.console import detect_console_encoding, locale
+
+
+def mock_raises_exception(error=Exception):
+    raise error
+
+
+@pytest.mark.parametrize('empty,filled', [
+    ['stdin', 'stdout'],
+    ['stdout', 'stdin']
+])
+def test_detect_console_encoding_from_stdout_stdin(monkeypatch, empty, filled):
+    # Ensures that when sys.stdout.encoding or sys.stdin.encoding is used when
```

Comment:
Can add a reference to the issue for this and the other new tests, i.e. `# GH 21552`